app/controllers/documents_controller.rb

```diff
@@ -0,0 +1,69 @@
+class DocumentsController < ApplicationController
+  def index
+      @documents = Document.all.order(created_at: :desc)
+  end
+
+  def new
+    @document = Document.new
+  end
+
+  def create
+    @document = Document.new
+    @document.title = params[:document][:title]
+    @document.description = params[:document][:description]
+    @document.keywords = params[:document][:keywords]
+    @document.user_id = current_user.id
+
+
+    if @document.save
+      flash[:notice] = "Document Uploaded!"
+      redirect_to documents_path
+    else
+      flash[:alert] = "There was an error saving the document. Please try again."
+      render :new
+    end
+  end
+
+  def show
+    @document = Document.find(params[:id])
+  end
+
+  def edit
+    @document = Document.find(params[:id])
+  end
+
+  def update
+    @document = Document.find(params[:id])
+    @document.assign_attributes(document_params)
```

Comment:
You could remove this line and change the if statement to `if @document.update(document_params)`. Update returns a boolean value.